extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -84,6 +88,9 @@ export class AmpAdApiHandler {
     this.iframe_ = iframe;
     this.is3p_ = is3p;
     this.iframe_.setAttribute('scrolling', 'no');
+    let iframeSrc = this.iframe_.getAttribute('src');
+    iframeSrc = iframeSrc.substring(0, iframeSrc.indexOf('#'));
+    this.useApi_ = endsWith(iframeSrc, '.html');
```

Comment:
sorry, i regretted. checking iframe src is a bad idea. because we allow remote bootstrap frame.

does `baseinstance instanceof AmpA4A` work?